apps/test/unit/templates/instructions/TopInstructionsTest.jsx

```diff
@@ -171,190 +171,217 @@ describe('TopInstructions', () => {
     expect(wrapper.find('Button')).to.have.lengthOf(0);
   });
 
```

Comment:
I'm not sure why this diff is such a nightmare -- I added the tests `displays StudentRubricView when rubrics tab is selected` and `passes displayTaRubricTab=true to TopInstructionsHeader if rubric is present`